### ISSUE TITLE
Publish 0.5.28

### ISSUE THREAD

**Issue by ngbinh** [trueaccord/ScalaPB#124]
Still not available in Maven Central yet: http://mvnrepository.com/artifact/com.trueaccord.scalapb/scalapb-runtime_2.11/0.5.28

**Comment by eiennohito** [comment]
It is available, actually:
http://repo1.maven.org/maven2/com/trueaccord/scalapb/scalapb-runtime_2.11/0.5.28/

Just mvnrepository is slow to index.

**Comment by ngbinh** [comment]
Sorry, I meant the sbt plugin:

```scala
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  ::          UNRESOLVED DEPENDENCIES         ::
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  :: com.trueaccord.scalapb#sbt-scalapb;0.5.28: not found
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
```

**Comment by thesamet** [comment]
I'll release 0.5.29 this weekend.

**Comment by thesamet** [comment]
Released 0.5.29.
